cw-relayer/go.mod

```diff
@@ -3,13 +3,11 @@ module github.com/ojo-network/cw-relayer
 go 1.19
 
 require (
-	github.com/CosmWasm/wasmd v0.0.0-00010101000000-000000000000
 	github.com/cosmos/cosmos-sdk v0.46.7
-	github.com/cosmos/go-bip39 v1.0.0
+	github.com/ethereum/go-ethereum v1.10.17
```

Comment:
We're going to need dependency update workflows directly to this branch

Reply:
would modify depandabot config to make updates against evm branch